components/AuthWrapper.js

```diff
@@ -42,26 +42,27 @@ export default function AuthWrapper({ Component, pageProps }) {
 
     if (isLoading) {
         return t`Loading...`;
-    } else if ((userDetailsError || datasetsError) && userDetails != 'user_not_found_in_adr') {
+    } else if ((userDetailsError || datasetsError)) {
+        console.log(userDetailsError)
         const invalidAuthError =
             userDetailsError && userDetailsError.message.includes('401')
             || datasetsError && datasetsError.message.includes('401')
+        const adrUserNotFound = userDetailsError && userDetailsError.message.includes('404')
```

Comment:
This is fine if it is enough that we check only the HTTP code. I thought it would be better to check the error message as well (e.g. check the 'message' value in JSON response, which is hidden here), but if the HTTP code is ok, then I'm ok with that. In the latter case we should delete the NAV-278 so Chas won't need to fix this.